fem/transfer.hpp

```diff
@@ -303,14 +433,25 @@ class L2ProjectionGridTransfer : public GridTransfer
       /// conservative left-inverse prolongation operation. This functionality
       /// is also provided as an Operator by L2Prolongation.
       void ProlongateTranspose(const Vector& x, Vector& y) const override;
+
+      void DeviceProlongateTranspose(const Vector& x, Vector& y) const;
+
+      /// Returns the inverse of an on-rank lumped mass matrix
+      void LumpedMassInverse(Vector& ML_inv) const;
+
       void SetRelTol(real_t p_rtol_) override;
       void SetAbsTol(real_t p_atol_) override;
+
    protected:
       /// Sets up the PCG solver (sets parameters, operator, and preconditioner)
       void SetupPCG();
-      /// Computes on-rank R and M_LH matrices.
+      void DeviceSetupPCG();
```

Comment:
The tolerances for `pcg` class member are set by `SetRelTol` and `SetAbsTol`, however the tolerances for `pcg_vea` are not. I think it makes sense for these methods set the tolerances for the `pcg_vea` as well.

Reply:
Sounds good to me.